### PR TITLE
DI-207: Base Sqlalchemy Model

### PR DESCRIPTION
**Pull request by aviral-nayya**
> **_SHIPPED NOTE:_**  The note content.

## As the author of this PR, I have:

- [ ] Manually tested my proposed changes and verified that there were no server errors or UI bugs
- [ ] Written a summary of the proposed changes as a PR comment and how to test them, if applicable ([example](https://github.com/Nayya-com/nayya-insurance/pull/327))
- [ ] If applicable, I have added documentation in [notion](https://www.notion.so/Product-and-Engineering-e965393fa4154611b4b008d9adb08e86) describing how the feature works
- [ ] Confirmed that automated tests are passing
- [ ] Assigned one or more [reviewers](https://www.notion.so/Code-Reviews-0c16b837cc4b418c904ed39691c193fe#2e862cf919f14c95937e4d373f218ad6)

### CODE REVIEW

File: app/base/model.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+@declarative_mixin
+class DbBaseModel:
+    """base model for SQLAlchemy Models"""
+
+    @declared_attr
+    def __tablename__(cls): # type: ignore
+        return f"{camel_to_snake_case(cls.__name__)}s"
+    
+    id = Column(UUID(as_uuid=True), default=lambda: str(uuid.uuid4()), primary_key=True)
+    created_at = Column(DateTime, default=datetime.now, nullable=False)
```
Comment (CamiloEVelasco) [inline app/base/model.py:19]: Maybe add `updated_at` to this as well?

**Comment by CamiloEVelasco** [comment]
Is there a ticket to add test suite and type checking github actions? If so, now seems like a good time to tackle it so the tests we're writing run here

**Review by jordan-clarkyy — APPROVED** [review]

**Review by jordan-clarkyy — APPROVED** [review]

File: app/base/__init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+# placeholder
```
Comment (CamiloEVelasco) [inline app/base/__init__.py:1]: Hopefully we can get rid of these once this is merged

**Review by CamiloEVelasco — APPROVED** [review]
Great work! We're moving at a rampant pace!

**Pull request merged**